pkg/controller/servicemesh/controlplane/htpasswd.go

```diff
@@ -51,6 +53,7 @@ func (r *controlPlaneInstanceReconciler) patchHtpasswdSecret(ctx context.Context
 	b64Password := base64.StdEncoding.EncodeToString([]byte(rawPassword))
 	b64Auth := base64.StdEncoding.EncodeToString([]byte(auth))
 
+	// b/c Keep rawPassword for previous release Kiali, Prometheus and Grafana patching
```

Comment:
Could you somehow append this information to the existing comment so that they are one coherent piece of information? And what "b/c" means?

Reply:
I googled an [Abbreviation](https://en.wiktionary.org/wiki/Appendix:Glossary#abbreviation) of [backward compatibility](https://en.wiktionary.org/wiki/backward_compatibility#English).

I will organize the comment.